.azure-pipelines.yml

```diff
@@ -0,0 +1,26 @@
+# https://github.com/actions/virtual-environments
+variables:
+  DEFAULT_NODE_VERSION: "12.x"
+
+pool:
+  vmImage: "windows-latest"
+
+stages:
+- stage: Unit_Tests_And_Linters
+  jobs:
+  - job: Linter
+    steps:
+    - template: azure-templates/node_setup_steps.yml
+    - script: npm run lint
+
+  - job: Unit_Tests
+    steps:
+    - template: azure-templates/node_setup_steps.yml
+    - script: npm run test
+
+- stage: Integration_Tests
+  jobs:
+  - job: Integration_Tests
+    steps:
+    - template: azure-templates/node_setup_steps.yml
+    - script: npm run e2e-test
```

Comment:
Generally speaking the windows driver does not install WinAppDriver on the first run of `npm install`. This might need to run the install again. WinAppDriver also needs special permissions. This was the problem with running the e2e tests in Travis.

Reply:
lets try to do what we can (run at least unit tests)

Reply:
the server we could try to download using Power Shell

Reply:
The app _will_ download. It's just the first time through the code hasn't been transpiled yet. The second time it downloads. It just won't run. I don't have a windows machine so I can't try it out.

Also, the unit tests were running in Travis.